app/main.py

```diff
@@ -1,3 +1,3 @@
 def count_occurrences(phrase: str, letter: str) -> int:
-    # write your code here
+    return phrase.lower().count(letter.lower())
```

Comment:
The function correctly counts the occurrences of a letter in a phrase, ignoring case. The implementation is correct.